iron-input.html

```diff
@@ -28,6 +28,15 @@
 
     <input is="iron-input" bind-value="{{myValue}}">
 
+`iron-input` of type "number" or "date" can use the `bind-value-as-number` property for two-way data binding.
```

Comment:
did you mean the "or date" here? What happens if i set a date to 3, does it just get converted magically?